nixos/modules/services/misc/klipper.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.klipper;
+  package = pkgs.klipper;
+  format = pkgs.formats.ini { mkKeyValue = generators.mkKeyValueDefault {} ":"; };
+in
+{
+  ##### interface
+  options = {
+    services.klipper = {
+      enable = mkEnableOption "Klipper, the 3D printer firmware";
+
+      settings = mkOption {
+        type = format.type;
+        default = { };
+        description = ''
+          Configuration for Klipper. See the <link xlink:href="https://www.klipper3d.org/Overview.html#configuration-and-tuning-guides">documentation</link>
+          for supported values.
+        '';
+      };
+
+      logDir = mkOption {
+        type = types.str;
+        default = "/var/log/klipper";
+        description = "Log directory of the daemon.";
+      };
+    };
+  };
+
+  ##### implementation
+  config = mkIf cfg.enable {
+    systemd.tmpfiles.rules = [
+      "d '${cfg.logDir}' - root root - -"
```

Comment:
`LogsDirectory=klipper` (if we can restrict `logDir` to `/var/log`, if we want it at all).

Reply:
Removed the need for this altogether